tests/helper/helper_interactive.go

```diff
@@ -0,0 +1,73 @@
+package helper
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"github.com/kr/pty"
+)
+
+//func RunInteractive(commonVar CommonVar, interVar Interactive) (string, error) {
+func RunInteractive(commonVar CommonVar, command []string, test func(*expect.Console, *bytes.Buffer) (bytes.Buffer, error)) (string, error) {
+
+	// tmpdir, _ := ioutil.TempDir("", "")
+	os.Chdir(commonVar.Context)
+	// fmt.Println(tmpdir)
+	// defer os.RemoveAll(tmpdir)
+
+	ptm, pts, err := pty.Open()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	term := vt10x.New(vt10x.WithWriter(pts))
+
+	c, err := expect.NewConsole(expect.WithStdin(ptm), expect.WithStdout(term), expect.WithCloser(pts, ptm))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer c.Close()
+
+	//cmd := exec.Command("odo", "init")
+	cmd := exec.Command(command[0], command[1:]...)
+	cmd.Stdin = c.Tty()
+	cmd.Stdout = c.Tty()
+	cmd.Stderr = c.Tty()
+	err = cmd.Start()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	buf := new(bytes.Buffer)
+	//var output string
+	res, err := test(c, buf)
```

Comment:
As you pass `buf` as a pointer, you could use the value of `buf` after `tests` returned, instead of getting a copy of it in `res`:

```
err := test(c, buf)
[...]
return buf.String(), err
```

Reply:
Ooh, yes, Thanks @feloy, I overlooked it.
I will update it.